gobblin-yarn/src/main/java/org/apache/gobblin/yarn/GobblinYarnAppLauncher.java

```diff
@@ -204,7 +204,7 @@ public class GobblinYarnAppLauncher {
 
   private final Optional<String> appMasterJvmArgs;
 
-  private final Path sinkLogRootDir;
+  private final String sinkLogRootDir;
```

Comment:
does anything need to change where it's used, now that this is a `String` and no longer a `Path`?

Reply:
Nope because we create a new Path using this string, but omit creating the path if it's null. https://github.com/apache/gobblin/blob/b08ad65948deedc00c94e828ba5a76c9578cb93c/gobblin-yarn/src/main/java/org/apache/gobblin/yarn/GobblinYarnAppLauncher.java#L379